### DISCUSSION TITLE
[select] 请允许开发者自行决定el-select等表单控件的keydown事件是否冒泡

### DISCUSSION THREAD

**Discussion by morowind** [element-plus/element-plus#17858] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `select`

## Description

我的应用中需要在表单上支持快捷键，通过监听document上的keydown事件实现，实现过程中发现，如果焦点在el-select，el-date-picker等控件上时，快捷键就会失效，应该是这些控件阻止了keydown的事件冒泡。现在我需要能够订阅到这些控件上冒出来的事件，建议在这些控件上添加一个开关属性，如果打开开关，keydowan事件就可以冒泡，允许用户自行处理。

### Additional comments

用户要求的关键功能，还望支持一下

**Comment by tolking** [comment]
有些按键不能够冒泡的。

比如：方向键和空格键会滚动页面；
Esc 不但能关闭 select 的弹窗也能够关闭 dialog；

或许你可以尝试通过监听事件实现

**Reply by morowind** [reply]
我明白一些特殊按键不能冒泡，但是为什么要拦截包含了这些特殊按键的组合键，比如：Ctrl + Enter ?
